modules/options-for-notebook-server-environments.adoc

```diff
@@ -314,15 +389,15 @@ a| * Python 3.9
 
 ifdef::upstream[]
 | RStudio Server
-| 2023.2 (Recommended)
+| 2023.2  (Recommended)
```

Comment:
Is it a necessary change?

Reply:
No - same as https://github.com/opendatahub-io/opendatahub-documentation/pull/248/files#r1567139211.

Reply:
Removed 2023.2 version for RStudio.